MC/run/ANCHOR/anchorMC.sh

```diff
@@ -126,7 +126,6 @@ echo_info "Substituting ALIEN_JDL_LPMPRODUCTIONTAG=$ALIEN_JDL_LPMPRODUCTIONTAG w
 ALIEN_JDL_LPMPRODUCTIONTAG=$ALIEN_JDL_LPMANCHORPRODUCTION
 
 if [[ $ALIEN_JDL_ANCHOR_SIM_OPTIONS == *"--tpc-distortion-type 2"* ]]; then
-  export O2DPG_ENABLE_TPC_DISTORTIONS=ON
```

Comment:
@sawenzel suppression of this line leads to disabling corrections in the reconstruction even if distortions were used in the digitization (this happened in the recent test of Mesut LHC25b8a_default_v4, while in LHC25b8a_default_v3 they were OK). 
I remember you did something else to detect distortion used in the digi. but I don't remember what and it seems it did not work. Shall we simply restore this line?